src/login/LoginPage.jsx

```diff
@@ -1,395 +1,318 @@
-          {this.props.resetPassword && !this.props.loginError ? <ResetPasswordSuccess /> : null}
-          <Form name="sign-in-form" id="sign-in-form">
-            <FormGroup
-              name="emailOrUsername"
-              value={this.state.emailOrUsername}
-              autoComplete="on"
-              handleChange={(e) => this.setState({ emailOrUsername: e.target.value, isSubmitted: false })}
-              handleFocus={this.handleOnFocus}
-              handleBlur={this.handleOnBlur}
-              errorMessage={this.state.errors.emailOrUsername}
-              floatingLabel={intl.formatMessage(messages['login.user.identity.label'])}
-            />
-            <PasswordField
-              name="password"
-              value={this.state.password}
-              autoComplete="off"
-              showRequirements={false}
-              handleChange={(e) => this.setState({ password: e.target.value, isSubmitted: false })}
-              handleFocus={this.handleOnFocus}
-              handleBlur={this.handleOnBlur}
-              errorMessage={this.state.errors.password}
-              floatingLabel={intl.formatMessage(messages['login.password.label'])}
-            />
-            <StatefulButton
-              name="sign-in"
-              id="sign-in"
-              type="submit"
-              variant="brand"
-              className="login-button-width"
-              state={submitState}
-              labels={{
-                default: intl.formatMessage(messages['sign.in.button']),
-                pending: '',
-              }}
-              onClick={this.handleSubmit}
-              onMouseDown={(e) => e.preventDefault()}
+          <PasswordField
+            name="password"
+            value={formFields.password}
+            autoComplete="off"
+            showScreenReaderText={false}
+            showRequirements={false}
+            handleChange={handleOnChange}
+            handleFocus={handleOnFocus}
+            errorMessage={errors.password}
+            floatingLabel={formatMessage(messages['login.password.label'])}
+          />
+          <StatefulButton
+            name="sign-in"
+            id="sign-in"
+            type="submit"
+            variant="brand"
+            className="login-button-width"
+            state={submitState}
+            labels={{
+              default: formatMessage(messages['sign.in.button']),
+              pending: '',
+            }}
+            onClick={handleSubmit}
+            onMouseDown={(event) => event.preventDefault()}
+          />
+          <Link
+            id="forgot-password"
+            name="forgot-password"
+            className="btn btn-link font-weight-500 text-body"
+            to={updatePathWithQueryParams(RESET_PAGE)}
+            onClick={trackForgotPasswordLinkClick}
+          >
+            {formatMessage(messages['forgot.password'])}
+          </Link>
+          {provider && (
+            <EnterpriseSSO provider={provider} />
```

Comment:
in the prev implementation, the `EnterpriseSSO` component was only being rendered in the `if (this.tpaHint)` and then the `provider` conditional, this is not true in here. Its only under the `provider` conditional here.

Any issues?

Reply:
As discussed in standup, this is a bug and will be resolved by @ahtesham-quraish

Reply:
okay

Reply:
it seems fixed but could you please test it locally to make sure if tpa_hint is working fine?

Reply:
I have tested it locally it dont see the issue but let me test it again